### ISSUE TITLE
[Bug]: When I look at bees my FPS reduces

### ISSUE THREAD

**Issue by EvaLinn** [Resourceful-Bees/ResourcefulBees#377]
### Bug Description

When I look at bees my FPS reduces

Mods that I have
![Screenshot_4](https://github.com/Team-Resourceful/ResourcefulBees/assets/83809307/847d4208-0ee7-4239-8cef-16fc65b38f0d)

### How to Reproduce?

Fly around the world
Bees are spawning
When I look at them I have lags and reduced FPS
If i will fly away where's no bees, I have good FPS

### Expected Behavior

Not to cause massive FPS drops when I near bees

### Version

resourcefulbees-forge-1.19.2-1.0.0-alpha.21

### Mod Loader Version

1.19.2 - Forge 43.2.8

### Logs or additional context

![Screenshot_2](https://github.com/Team-Resourceful/ResourcefulBees/assets/83809307/9b7f29cf-8ba1-4a98-bb71-58ed151afbe8)
![Screenshot_3](https://github.com/Team-Resourceful/ResourcefulBees/assets/83809307/0410c14b-dc75-4801-9f5d-ac406eefc2a2)

### Acknowledgement

- [X] I agree that I read the terms at the top of the page.

**Comment by Epic428** [comment]
Yes I’m very aware that this happens. There’s very little I can do to fix it outside of writing a new rendering system for Minecraft. The number of cubes each bee model has is part of the issue. However reductions to those means the bees don’t look fancy. Unfortunately for now it is something we have to deal with. Another modder is working on a rendering system that we should be able to depend on which would drastically improve the performance. It would only be 1.20+ and is still very experimental

**Comment by EvaLinn** [comment]
Is there any way to change config of spawn? I reduced the amount of nests in overworld but there's still a lot of bees. I'd like to make them rare, so I won't lag always XD

**Comment by Epic428** [comment]
You can make a datapack and configure bee spawns with the biome modifiers and spawn data. Here’s the files for the default bees if you want to modify them and override

https://github.com/Team-Resourceful/ResourcefulBees/tree/1.19/forge/src/main/resources/data/resourcefulbees/forge/biome_modifier
